libsel4/tools/sel4_idl.dtd

```diff
@@ -26,23 +26,23 @@
 <!ATTLIST method manual_name CDATA #IMPLIED>
 <!ATTLIST method manual_label CDATA #IMPLIED>
 
-<!ELEMENT param ANY>
+<!ELEMENT param (#PCDATA|description|error)*>
 <!ATTLIST param type CDATA #REQUIRED>
 <!ATTLIST param name CDATA #REQUIRED>
 <!ATTLIST param dir CDATA #REQUIRED>
 <!ATTLIST param description CDATA #IMPLIED>
 
-<!ELEMENT error ANY>
+<!ELEMENT error (#PCDATA|description)*>
 <!ATTLIST error name CDATA #REQUIRED>
 <!ATTLIST error description CDATA #IMPLIED>
 
-<!ELEMENT brief ANY>
+<!ELEMENT brief (#PCDATA|texttt|docref|shortref|autoref|obj)*>
 
-<!ELEMENT description ANY>
+<!ELEMENT description (#PCDATA|texttt|docref|shortref|autoref|obj)*>
 
-<!ELEMENT return ANY>
+<!ELEMENT return (#PCDATA|texttt|docref|shortref|autoref|obj|errorenumdesc)*>
```

Comment:
Wasn't sure if `errorenumdesc` belonged in the other ones or not it wasn't used anywhere but `return`.

Reply:
I think it makes sense to restrict it to this one. We can always add it elsewhere if we decide it is necessary.